runtime/compiler/optimizer/IdiomTransformations.cpp

```diff
@@ -846,44 +846,50 @@ usedInLoopTest(TR::Compilation *comp, TR::Node *loopTestNode, TR::SymbolReferenc
    }
 
 static bool
-indexContainsArray(TR::Compilation *comp, TR::Node *index, vcount_t visitCount)
+indexContainsArray(TR::Compilation *comp, TR::Node *index, vcount_t visitCount, bool shouldTrace)
```

Comment:
Can we make these private member functions and just use `trace()` to avoid the complexity of additional parameters? Similarly for other functions modified.

Reply:
Fixed additional parameter in 4a94f30
As we talked today, they weren't made private member functions because they are called from other functions.